scripts/install_stacks.sh

```diff
@@ -131,8 +145,8 @@ ConditionFileIsExecutable=/usr/local/bin/stacks-node
 ConditionPathExists=/stacks-blockchain/
 
 [Service]
-ExecStart=/bin/sh -c "/usr/local/bin/stacks-node start --config=/etc/stacks-blockchain/Config.toml >> /stacks-blockchain/miner.log 2>&1"
-ExecStartPost=/bin/sh -c "umask 022; sleep 2 && pgrep -f \"/usr/local/bin/stacks-node start --config=/etc/stacks-blockchain/Config.toml\" > /run/stacks-blockchain/stacks.pid"
+ExecStart=/bin/sh -c "/usr/local/bin/stacks-node start --config /etc/stacks-blockchain/Config.toml >> /stacks-blockchain/miner.log 2>&1"
+ExecStartPost=/bin/sh -c "umask 022; sleep 2 && pgrep -f \"/usr/local/bin/stacks-node start --config /etc/stacks-blockchain/Config.toml\" > /run/stacks-blockchain/stacks.pid"
```

Comment:
this is ok for now - but i think we can reduce the complexity of this unit with changes to the shutdown process of the stacks-node binary:
https://github.com/stacks-network/stacks-core/blob/master/contrib/init/stacks.service

Reply:
resolving. this is good enough for now, i'll create an issue to update so it's closer to https://github.com/stacks-network/stacks-core/blob/master/contrib/init/stacks.service